rumydata/_base.py

```diff
@@ -229,7 +229,8 @@ def _flatten_exceptions(cls, error):
         checking to see if any of those errors contain additional errors, then
         continuing to recurse until all errors have been yielded.
         """
```

Comment:
Some of the new tests were not expected to fail, but the series of method calls eventually ends up calling `_flatten_exceptions`, this will check to make sure an error exists before attempting to call `_errors`